Blueprints/BlueprintDefinitions/vs2022/CustomRuntimeFunction-FSharp/blueprint-manifest.json

```diff
@@ -1,8 +1,8 @@
 {
   "display-name": "Custom Runtime Function",
   "system-name": "CustomRuntimeFunction",
-  "description": "Use Lambda Custom Runtime feature to build Lambda functions using .NET 7.",
-  "sort-order": 400,
+  "description": "Use Lambda Custom Runtime feature to build Lambda functions using .NET 8.",
+  "sort-order": 125,
```

Comment:
I changed the sort order to make the custom runtime blueprints more prominent now that we are targeting .NET 8.